server/src/main/java/org/elasticsearch/cluster/coordination/Coordinator.java

```diff
@@ -335,6 +341,15 @@ public Coordinator(
         this.leaderHeartbeatService = leaderHeartbeatService;
         this.compatibilityVersions = compatibilityVersions;
         this.clusterService = clusterService;
+        this.clusterFormationClusterStateView = new ClusterFormationFailureHelper.ClusterFormationClusterStateView(
+            null,
+            null,
+            0,
+            0,
+            null,
+            null,
```

Comment:
Could we use an empty map and `org.elasticsearch.cluster.coordination.CoordinationMetadata.VotingConfiguration#EMPTY_CONFIG` rather than nulls here. Bit of a pain we can't fill in the local disco node too but we don't know it at this point in startup.